src/services/color/color_palette.ts

```diff
@@ -1,98 +1,92 @@
-import { rgbDef } from './color_types';
-import { hexToRgb } from './hex_to_rgb';
+import chroma from 'chroma-js';
 
-/**
- * Create the color object for manipulation by other functions
- */
-class Color {
-  collection: rgbDef;
-  text: string;
-
-  constructor(public r: number, public g: number, public b: number) {
-    this.collection = [r, g, b];
-    this.text = createHex(this.collection);
-  }
-}
+export const MID_COLOR_STOP = '#EBEFF5';
 
 /**
- * This function takes a color palette name and returns an array of hex color
- * codes for use in UI elements such as charts.
- *
- * @param {string} hexStart The beginning hexadecimal color code
- * @param {string} hexEnd The ending hexadecimal color code
- * @param {number} len The number of colors in the resulting array (default 10)
- * @returns {Array} Returns an array of hexadecimal color codes
+ * This function takes an array of colors and returns an array of interpolated
+ * colors based on the number of steps/len needed for use in UI elements such as charts.
+ * Derived from https://github.com/gka/palettes
```

Comment:
```suggestion
 * Derived from https://github.com/gka/palettes (Unlicensed)
```

Just for good measure